diagnostics-app/src/app/telemetry/cards/cpu/cpu.component.ts

```diff
@@ -12,6 +12,25 @@ import { CpuInfo, LogicalCpuInfo } from '@common/dpsl';
 import { TelemetryService } from 'src/app/core/services/telemetry.service';
 import refreshIntervals from '../../../core/config/data-refresh-intervals';
 
+const defaultCpuInfo: CpuInfo = {
+  numTotalThreads: 0,
+  architecture: 'unknown',
+  physicalCpus: [
+    {
+      modelName: 'unknown',
+      logicalCpus: [
+        {
+          maxClockSpeedKhz: 0,
+          scalingMaxFrequencyKhz: 0,
+          scalingCurrentFrequencyKhz: 0,
+          idleTimeMs: 0,
+          acStates: [],
```

Comment:
Can you check the field name?

From telemetry.idl  (https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/chromeos/extensions/api/telemetry.idl?q=telemetry.idl), I think the name is `cStates` not `acStates`.

Reply:
Thank you for catching this bug.

It turned out the CpuInfo defined in "dpsl.ts" had a few incorrect field names and types, so I first fixed the definition in another PR: https://github.com/GoogleChromeLabs/cros-sample-telemetry-extension/pull/51, and then I updated the default CpuInfo here.